src/main/java/dev/unnm3d/rediseconomy/currency/CurrencyWithBanks.java

```diff
@@ -333,8 +332,7 @@ public CompletionStage<List<ScoredValue<String>>> getOrderedBankAccounts() {
      * @return A map of bank owners
      */
     public CompletionStage<Map<String, String>> getRedisBankOwners() {
```

Comment:
Please remove all the methods in this class that contains only one line. Replace the lines that are calling those methods with the methods from EconomyStorage